src/components/Toasts/TransactionToast.styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components/macro";
+
+export const Container = styled.div`
+  color: ${(props) => props.theme.colors.white};
+  background-color: ${(props) => props.theme.colors.darkGrey} !important;
```

Comment:
There are a few `!important` rules in here - as I'm sure you know this is generally discouraged - are you overriding some built-in-style or something?

I think if we need to use `!important` there should be a comment to explain exactly why - I feel like it should be very rare with styled components.